plugins/rias/ls.rias.js

```diff
@@ -186,6 +186,9 @@
 
 		if(buggySizes && elem.nodeName.toUpperCase() == 'IMG'){
 			elem.removeAttribute(config.srcsetAttr);
+			// at least on Safari 9, it looks like if srcset is set, it is preferred.
+			// Even though buggySizes is true. let's remove it to make sure src is used.
+			elem.removeAttribute('srcset');
```

Comment:
I don't want to take this right now. There are many different versions of Safari 9.x.x and many of them had different bugs. Removing `srcset` does too much. 

I assume you tested with a very early 9.0.x version that always takes the first image candidate, if it does not understand its content?